resources/prosody-plugins/mod_muc_breakout_rooms.lua

```diff
@@ -0,0 +1,466 @@
+-- This module is added under the main virtual host domain
```

Comment:
@damencho I leave this module for you to review :-)

Reply:
@saghul Hum, while we were discussing this we assumed that breakout rooms are created under the main muc module ... this confuses me as this configures a new muc module.
We can handle here the room creation and drop the jicofo logic maybe ....

Reply:
Good thoughts @damencho. Breakout rooms are somewhat special as they require focus for media handling but not for the start room control. I first tried to handle this on Prosody only without touching jicofo by using the dedicated muc component. It turned out that I had to make jicofo aware of the breakout rooms. Now that jicofo handles starting breakout rooms correctly, I see a chance of getting rid of the specific muc, only distinguishing the breakout rooms from main rooms by the uuid or eventually by some specific name prefix as proposed by @saghul. This would simplify the Prosody module. What do you think?

Reply:
Removed extra muc component definition for breakout rooms. They are using the main muc domain now. Breakout rooms are distinguished by a #breakout_uuid suffix.